charts/centralidp/values.yaml

```
@@ -63,7 +63,8 @@ keycloak:
     # Enable TLS configuration for the host defined at `ingress.hostname` parameter;
     # TLS certificates will be retrieved from a TLS secret with name: `{{- printf "%s-tls" .Values.ingress.hostname }}`;
     # Provide the name of ClusterIssuer to acquire the certificate required for this Ingress.
-    annotations: {}
+    annotations:
+      {}
```

were those format changes on purpose? I'd refrain from doing those while release a stable version, would be better to them for an rc version so that it's tested in case
```suggestion
    annotations: {}
```